handbook/governance/code-of-ethics.md

```diff
@@ -0,0 +1,43 @@
+# dOrg Code of Ethics
+
+To ensure that dOrg remains an exceptional place to work and a valuable service provider to our clients & partners, we must hold ourselves to high ethical standards such as those outlined here.
+
+## Avoided Behaviors
+The following behaviors should be avoided:
+
+* Accepting work from dOrg clients/partners independently without paying a referral fee (20%) to dOrg for sourcing said work
+* Deletion or malicious alteration of communications/comments/messages on dOrg channels except, in cases, where ethical violations have occurred and such action has been approved by dOrg moderators
+* Engagement in abuse, discrimination, disrespect, harassment, or promotion of said or otherwise offensive content against/to any prospective/past/current dOrg client/partner/builder
+* Exertion of personal copyright over any work prepared for, paid for by, or otherwise commissioned by dOrg, or any past/current dOrg client/partner
+* Obstruction of dOrg operations through 'down staking' of DAO proposals except, in cases, where the proposer has admitted that a wish to rescind their proposal and/or a significant enough margin of voters have expressed a wish to change their votes such that the outcome (passing or failing) of the proposal would be otherwise changed
+* Participation in any strike, slowdown, sabotage, or work stoppage in retaliation for the action(s) or inaction of any prospective/past/current dOrg client/partner/builder
+* Public defamation of any prospective/past/current dOrg client/partner/builder without prior authorization and/or through unofficial channels
+
+## Enforcement
+dOrg follows the principle of restorative justice when avoided behaviors occur but builds in certain safeguards to protect the collective as a whole.
+
+1. When an avoided behavior seems to have occured, the offender(s) should first be contacted directly in order to reach an accord. Perhaps, the offender(s) made (a) mistake(s) and are/is prepared to immediately rectify it themselves. 
+2. If this is not possible, those affected should contact dOrg moderators to mitigate any immediate damage that may have occured and attempt to mediate between the victim(s) & the offender(s). A mediation should include sharing experiences of what happened, discusssion of who was harmed by the behavior and how, the creation of a consensus for what the offender(s) can do to repair the harm from the offense(s), and the enaction of said consensus.
+3. If a mediation is successful, then the DAO should pass a 'censure' proposal of the offender(s) which includes a record (private to dOrg members) of the mediation to be reviewed if/when future offense(s) occur. After 'censure' proposals pass or fail, the offender(s) are/is considered to be then in good standing with dOrg and no further action is neccesary.
```

Comment:
I don't think there should be a DAO proposal in cases where the parties reach accord. There  should only be a DAO proposal if there is a DAO-level action being taken like slashing rep or deactivating the member as in (4)

Reply:
I agree that passing a proposal may be un-necessary here, but there should be a record still.

Reply:
I removed the 'censure' proposal but kept the part about keeping a record. This may mean something like a pinned Discord message on the quality engagement channel linking to it.